### PR TITLE
remove sponsors and signup options from menu

### LINKED ISSUE #24: remove/hide unwanted stuff
From main menu: remove
1. "sign-up" (we add managers manually via admin, sponsors/CFP/tickets are managed by external means).
2. "sponsors" (we wil manage through admin)
3. Login: this is up for vote: depends if we want to manage the site from production or have a separate staging site.

Probably declined:
* Temporary remove sponsors section from homepage and the whole sidebar. That's because I wanted to go live without them and add later
(Since we did not go live in time, maybe better to just fix all the issues with sponsors section)

### PR DESCRIPTION
**Pull request by AmitAronovitch**
fixes #24

### CODE REVIEW

**Comment by AmitAronovitch** [comment]
Before merging, please let me know what you think re: removing the "login" option:
Advantage: UI element which serves only 5 out of who-knows-how-many users should not be there.
Disadvantage: We'll have to do all management in a separate "staging site"

File: ilpycon/sitetrees.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
       item('Blog', 'pinax_blog:blog'),
       item('Code of Conduct', 'pinax_pages:pages_page "en/code-of-conduct/"'),
       item('Venue', 'pinax_pages:pages_page "en/venue/"'),
-      item('Sponsors', '#', children=[
+      item('Sponsors', '#', in_menu=False, children=[
```
Comment (shaib) [inline ilpycon/sitetrees.py]: ... Then why leave it here at all?

**Comment by AmitAronovitch** [comment]
based on feedback: 
* hide the login button as well (configurable in settings)
* removed the sponsors menu from sitetrees

**Comment by shaib** [comment]
LGTM

**Pull request merged**